config/300-deployment.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: freeze-daemon
+  namespace: knative-serving
+spec:
+  selector:
+    matchLabels:
+      name: freeze-daemon
+  template:
+    metadata:
+      labels:
+        name: freeze-daemon
+    spec:
+      serviceAccountName: freeze-tokenreview # TODO: swap for account with just tokenreview
```

Comment:
is this todo still valid?

Reply:
I think it's probably already been done (and this just got C&Ped in error). Currently, the clusterrole for this service is only able to create tokenreviews and subjectaccessreviews.